opty/direct_collocation.py

```diff
@@ -654,9 +656,86 @@ class ConstraintCollocator(object):
     - o : number of instance constraints
     - nN + qN + r + s : number of free variables
     - n(N - 1) + o : number of constraints
+    - Some of the attributes are explained in more detail under Parameters below.
```

Comment:
This line does not belong in the list, but you can add it in the notes section as a normal paragraph line.